buildSoup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -eu
+# Generate Soups
+
+DEPLOY_DOC=${DEPLOY_DOC:-soup}
+TRAVIS_REPO_SLUG=${TRAVIS_REPO_SLUG:-owner/repo}
+APP="${TRAVIS_REPO_SLUG#*/}"
+GO111MODULE=on
+
+if [ -z "${TRAVIS_TAG+x}" ]; then
+  TRAVIS_TAG=0.0.0
+fi
+if [ -d ${DEPLOY_DOC} ]; then
+  rm -rf ./${DEPLOY_DOC}
```

Comment:
if we push to "docs" this deletion might have impacts on the generation of the openapi documentation in the other step